server/server.go

```diff
@@ -331,30 +317,15 @@ func NewServer(userConfig UserConfig, config Config) (*Server, error) {
 	}
 
 	underlyingRouter := mux.NewRouter()
-	router := &Router{
-		AtlantisURL:               parsedURL,
-		LockViewRouteIDQueryParam: LockViewRouteIDQueryParam,
-		LockViewRouteName:         LockViewRouteName,
-		ProjectJobsViewRouteName:  ProjectJobsViewRouteName,
-		Underlying:                underlyingRouter,
-	}
-
-	var projectCmdOutputHandler handlers.ProjectCommandOutputHandler
-	// When TFE is enabled log streaming is not necessary.
-
-	if userConfig.TFEToken != "" {
-		projectCmdOutputHandler = &handlers.NoopProjectOutputHandler{}
-	} else {
-		projectCmdOutput := make(chan *models.ProjectCmdOutputLine)
-		projectCmdOutputHandler = handlers.NewInstrumentedProjectCommandOutputHandler(
-			projectCmdOutput,
-			commitStatusUpdater,
-			router,
-			logger,
-			statsScope.Scope("api"),
-		)
-	}
+	router := NewRouter(underlyingRouter, parsedURL)
 
+	projectCmdOutputHandler := handlers.NewProjectCommandOutputHandler(
+		userConfig,
+		statsScope.Scope("api"),
```

Comment:
```suggestion
		statsScope,
```
nit: api has already been added to the scope in the initializer method here: https://github.com/lyft/atlantis/blob/080fc925c63ecfdf4b39fbe3f6a82229df62025d/server/handlers/instrumented_project_command_output_handler.go#L30

Reply:
Good call, i must have missed it.